.github/workflows/test-google-cloud-platform-install.yaml

```diff
@@ -0,0 +1,204 @@
+#
+# This is an e2e test to deploy PostHog on Google Cloud Platform using Helm.
+#
+# TODO:
+# - run k8s spec test
+# - run action only when necessary
+#
+name: e2e - Google Cloud Platform (install)
+
+on: push
+
+jobs:
+  gcp-install:
+    runs-on: ubuntu-20.04
+    if: github.repository == 'PostHog/charts-clickhouse'
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up GCP Cloud SDK
+      uses: google-github-actions/setup-gcloud@master
+      with:
+        project_id: ${{ secrets.GCP_PROJECT_ID }}
+        service_account_key: ${{ secrets.GCP_SA_KEY }}
+        export_default_credentials: true
+
+    - name: Install doctl
+      uses: digitalocean/action-doctl@v2
+      with:
+        token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
+
+    - name: Declare variables that we can share across steps
+      id: vars
+      run: |
+        TEST_NAME="helm-test-e2e-gcp-install-$(git rev-parse --short HEAD)"
+        echo "::set-output name=k8s_cluster_name::${TEST_NAME}"
+        echo "::set-output name=dns_record::${TEST_NAME}"
+        echo "::set-output name=fqdn_record::${TEST_NAME}.posthog.cc"
+
+    - name: Deploy a new k8s cluster
+      id: k8s_cluster_creation
+      run: |
+        gcloud container clusters create \
+          ${{ steps.vars.outputs.k8s_cluster_name }} \
+          --project ${{ secrets.GCP_PROJECT_ID }} \
+          --region europe-west4 \
```

Comment:
lets use `us-central-1` as that's cheaper and we don't care where we deploy. See https://cloud.google.com/products/calculator#id=ee71f5f9-4a0a-4caa-a122-e9389cdb8f3e